src/Microsoft.DotNet.Darc/src/Darc/Options/CloneCommandLineOptions.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using CommandLine;
+using Microsoft.DotNet.Darc.Operations;
+
+namespace Microsoft.DotNet.Darc.Options
+{
+    [Verb("clone", HelpText = "clone a remote repo and all of its dependency repos")]
+
+    internal class CloneCommandLineOptions : CommandLineOptions
+    {
+        [Option('l', "local", HelpText = "Clone all repos that this local repository declares a reference on.")]
```

Comment:
Local in the context of the other commands typically means "don't touch external resources".  I would actually just remove this option and imply it if repo and version is not passed in.

Reply:
Removed as suggested.